src/AutoRest.CSharp/Mgmt/Generation/ResourceGroupExtensionsWriter.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Output.Models;
+using Azure.Core.Pipeline;
+
+namespace AutoRest.CSharp.Generation.Writers
+{
+    internal class ResourceGroupExtensionsWriter
+    {
+        public void WriteExtension(CodeWriter writer, ResourceGroupExtensions resourceGroupExtensions, IDictionary<OperationGroup, ArmResource> armResources)
+        {
+            var cs = resourceGroupExtensions.Type;
+            var @namespace = cs.Namespace;
+            writer.UseNamespace(@"System");
+            writer.UseNamespace(@"Azure.ResourceManager.Core");
```

Comment:
You don't need to add these namespaces here. You can use typeOf instead of using string for the types.

Reply:
Done